libraries/APM_Control/AR_AttitudeControl.cpp

```diff
@@ -626,16 +626,14 @@ float AR_AttitudeControl::get_throttle_out_speed(float desired_speed, bool motor
     // set PID's dt
     _throttle_speed_pid.set_dt(dt);
 
-    // calculate base throttle (protect against divide by zero)
-    float throttle_base = 0.0f;
+    // update speed controller feed forward from cruise_speed and cruise_throttle values
     if (is_positive(cruise_speed) && is_positive(cruise_throttle)) {
-        throttle_base = desired_speed * (cruise_throttle / cruise_speed);
+        _throttle_speed_pid.ff(cruise_throttle / cruise_speed);
```

Comment:
this is directly setting the param, which is abit odd.

Reply:
Txs for the feedback.  I agree it's odd and the only reason I did this is because I can't directly overwrite just the _pid_info.FF.

Perhaps a cleaner alternative is to write an AR_AttitudeControl::get_steering_rate_pid_info() method that maintains a copy of the _throttle_speed_pid.pid_info and fills in the FF field with the original FF = this "throttle_base".

Another alternative would be to add an accessor to AC_PID that allows an external accessor to overwrite the pid_info.FF field.

Reply:
With the latest change I'm no longer updating the parameter values.